internal/gcs/container.go

```diff
@@ -227,3 +230,28 @@ func (c *Container) waitBackground() {
 	log.G(ctx).Debug("container exited")
 	oc.SetSpanStatus(span, err)
 }
+
+// UpdateContainerMemory makes a modify request on the given cow container to modify the memory size
+// Internally, this will modify the container's job object memory limits
+func UpdateContainerMemory(ctx context.Context, c cow.Container, newMemorySizeInBytes uint64) error {
```

Comment:
Talked offline but as this takes in a `cow.Container` parameter it feels out of place in the gcs package

Reply:
I think I'm just going to remove this from this PR for now and move the function in a later PR so it makes more sense.